config/initializers/constants.rb

```diff
@@ -0,0 +1,2 @@
+# Hold Google maps API key
+GOOGLE_MAPS_API_KEY = Rails.application.secrets.GOOGLE_MAPS_API_KEY
```

Comment:
I apologize, it was at the beginning of this year and I don't recall the specific details. I do remember making a pull request from the side of the road using my phone.